source/whats-new.txt

```diff
@@ -21,6 +22,28 @@ Learn what's new in:
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+.. _version-4.7.1:
+
+What's New in 4.7.1
+-------------------
+
+The 4.7.1 driver patches a potential data corruption bug in
+Client-Side Field Level Encryption (CSFLE) and Queryable Encryption (QE)
+that occurs when rotating :ref:`Data Encryption Keys <csfle-key-architecture>`
+(DEKs) encrypted with a :ref:`Customer Master Key <csfle-key-architecture>` (CMK)
+hosted on Google Cloud Key Management Service (GCP KMS) or Azure
+Key Vault.
+
+To avoid potential data corruption when rotating DEKs encrypted with a CMK
+hosted on GCP KMS or Azure Key Vault, upgrade to version 4.7.1 or later of the
+driver before using the ``RewrapManyDataKey`` method.
```

Comment:
Suggestion: until mention of RewrapManyDataKey method, all the info is redundant. Perhaps a short sentence about how the RewrapManyDataKey method causes the corruption in the previous paragraph is better.

Reply:
Agree, second paragraph is redundant. Moved information on RewrapManyDataKey to preceding paragraph, and added some information on how the bug specifically corrupts your data (lose DEKs).